### PR TITLE
Content sync and updated to MD

### PR DESCRIPTION
**Pull request by Vallejoanderson**
In Line 16, **Note** has been changed to **Nota**.

### CODE REVIEW

**Review by Graywolf9 — DISMISSED** [review]
Todo se ve correcto

File: files/es/web/api/battery_status_api/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Vea también: [El ejemplo en las especificaciones](https://dev.w3.org/2009/dap/system-info/battery-status.html#introduction).
+
+## Especificaciones
+
+{{page("/en-US/docs/Web/API/BatteryManager","Specifications")}}
+
+## Compatibilidad del Navegador
+
+{{page("/en-US/docs/Web/API/BatteryManager","Browser_compatibility")}}
```
Comment (Graywolf9) [inline files/es/web/api/battery_status_api/index.md:82]: Recordar que estamos quitando la macro `page` y tomar como ejemplo en inglés que `{{Specifications}}`

Comment (Vallejoanderson) [reply files/es/web/api/battery_status_api/index.md]: Listo, corregido @Graywolf9

File: files/es/web/api/battery_status_api/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+title: API de Estado de Bateria
+slug: Web/API/Battery_Status_API
+tags:
+  - API
+  - Apps
+  - Firefox OS
+  - Mobile
+translation_of: Web/API/Battery_Status_API
+original_slug: WebAPI/Estado_de_Bateria
```
Comment (JuanVqz) [inline files/es/web/api/battery_status_api/index.md:10]: vi estó en la doc en Inglés, debemos agregarlo?
```suggestion
original_slug: WebAPI/Estado_de_Bateria
browser-compat: api.BatteryManager
```

File: files/es/web/api/battery_status_api/index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+{{DefaultAPISidebar("Battery API")}}{{deprecated_header}}
+
+La **API de Estado de Batería**, también conocida como "**Battery API**", provee información acerca del sistema de carga de la batería y permite notificar mediante eventos que son enviados cuando el nivel de la batería cambia. Este puede ser usado para ajustar el uso de recursos por parte de una aplicación y evitar un gasto innecesario de energía cuando la batería esta baja o para guardar cambios en un archivo antes de que la batería se agote y así prevenir perdida de información.
+
+> **Nota:** Esta API _no está disponible_ en los [Web Workers](/es/docs/Web/API/Web_Workers_API) (no está expuesta en {{domxref("WorkerNavigator")}}).
+
+## Interfaces
+
+- {{domxref("BatteryManager")}}
+  - : Provee informacion acerca del nivel de carga de batería del sistema.
```
Comment (JuanVqz) [inline files/es/web/api/battery_status_api/index.md:21]:
```suggestion
  - : Provee información acerca del nivel de carga de batería del sistema.
```

File: files/es/web/api/battery_status_api/index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  });
+  function updateDischargingInfo(){
+    console.log("Tiempo de descarga de la batería: "
+                 + battery.dischargingTime + " segundos");
+  }
+
+});
+```
+
+Vea también: [El ejemplo en las especificaciones](https://dev.w3.org/2009/dap/system-info/battery-status.html#introduction).
```
Comment (JuanVqz) [inline files/es/web/api/battery_status_api/index.md:74]:
```suggestion
Vea también: [El ejemplo en las especificaciones](https://w3c.github.io/battery/).
```
El recurso fue movido a esta dirección

**Review by JuanVqz — APPROVED** [review]
unos pequeños cambios, si estas de acuerdo @Vallejoanderson...

File: files/es/web/api/battery_status_api/index.md (hunk 2 above)
Comment (Vallejoanderson) [reply files/es/web/api/battery_status_api/index.md:10]: No se a que se refiere exactamente, pero creo que lo agregamos por si acaso.

Comment (cw118) [reply files/es/web/api/battery_status_api/index.md:10]: If I may interject (still in English for now, unfortunately), this `browser-compat` line should be present in the front-matter of a file when using the `{{Compat}}` macro (it's not *strictly* necessary but it's good practice). This method is **preferred** to writing it as `{{Compat("api.BatteryManager")}}`.

**Comment by cw118** [comment]
If I may interject (still in English for now, unfortunately), the `browser-compat` line should be present in the front-matter of a file when using the `{{Compat}}` macro (it's not *strictly* necessary but it's good practice). This method is **preferred** to writing it as `{{Compat("api.BatteryManager")}}`.

**Comment by JuanVqz** [comment]
> If I may interject (still in English for now, unfortunately), the `browser-compat` line should be present in the front-matter of a file when using the `{{Compat}}` macro (it's not _strictly_ necessary but it's good practice). This method is **preferred** to writing it as `{{Compat("api.BatteryManager")}}`.

We want to follow the best practice. Do you know what it does?

Are you saying we should write it like the code below?

```md
---
title: title
slug: slug
...
browser-compat: {{Compat("api.BatteryManager")}}
```

I saw this PR, https://github.com/mdn/translated-content/pull/3966/files they added it as `browser-compat: html.elements.input.input-date
`

**Comment by cw118** [comment]
Hi @JuanVqz, sorry for the delay, and I realized I didn't word that clearly. I thought I'd explain about the compatibility data since I noticed Vallejoanderson ask about it — in short, I meant to say that the way `browser-compat` is written **now** in this PR (after Vallejoanderson's corrections) is **good/correct** :) 

<details>
<summary>The other/"older" way would be to write it like this, **without** any `browser-compat` line in the front matter:</summary>

```
---
title: API de Estado de Bateria
slug: Web/API/Battery_Status_API
tags:
  - API
  - Apps
  - Firefox OS
  - Mobile
translation_of: Web/API/Battery_Status_API
original_slug: WebAPI/Estado_de_Bateria
---

...

## Compatibilidad del Navegador

{{Compat("api.BatteryManager")}}
```
</details>

However, the front matter method, with `browser-compat: api.BatteryManager` and `{{Compat}}`, is what's **usually preferred/considered better practice**. They've been working on moving compat data into front matter on the mdn/content side, so while some are written as the "old method" above, the newer one (`browser-compat: something.here`) is what they're working on changing those to. Either way, with translated-content, we tend to follow in the en-US file's footsteps by writing the `{{Compat}}` however it's written in the English file.

I believe both ways have the same effect and rely on the [mdn/browser-compat-data repository](https://github.com/mdn/browser-compat-data). According to [this page](https://developer.mozilla.org/en-US/docs/MDN/Structures/Compatibility_tables#inserting_the_data_into_mdn_pages), the `{{Compat("api.BatteryManager")}}` method is sometimes used in order to include multiple compatibility tables; otherwise, we tend to favor the front matter way unless the English file doesn't use it. 

Welp, this is long... Main thing is that the way this PR currently has the browser-compat is perfect! I hope the rest of this makes sense, but if not, I've joined the Matrix chat so we can also talk there :)

File: files/es/web/api/battery_status_api/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  });
+  function updateDischargingInfo(){
+    console.log("Tiempo de descarga de la batería: "
+                 + battery.dischargingTime + " segundos");
+  }
+
+});
+```
+
+Vea también: [El ejemplo en las especificaciones](https://w3c.github.io/battery/).
```
Comment (cw118) [inline files/es/web/api/battery_status_api/index.md:75]: I might be wrong, but the en-US file has a different link:

```suggestion
Vea también: [El ejemplo en las especificaciones](https://www.w3.org/TR/battery-status/#examples).
```

File: files/es/web/api/battery_status_api/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
+---
+title: API de Estado de Bateria
+slug: Web/API/Battery_Status_API
+tags:
+  - API
+  - Apps
+  - Firefox OS
+  - Mobile
+translation_of: Web/API/Battery_Status_API
+original_slug: WebAPI/Estado_de_Bateria
```
Comment (cw118) [inline files/es/web/api/battery_status_api/index.md:10]: Missing a slash in the original slug:

```suggestion
original_slug: Web/API/Estado_de_Bateria
```

File: files/es/web/api/battery_status_api/index.md (hunk 5 above)
Comment (Vallejoanderson) [reply files/es/web/api/battery_status_api/index.md:75]: Thank you for notice it, @cw118  ;)

File: files/es/web/api/battery_status_api/index.md (hunk 6 above)
Comment (Vallejoanderson) [reply files/es/web/api/battery_status_api/index.md]: Nice catch @cw118. Thank you.

**Comment by JuanVqz** [comment]
@Vallejoanderson Gracias por tu actividad en la comunidad. Eres de enorme ayuda!

**Pull request merged**